interface/login/login.php

```diff
@@ -95,11 +95,14 @@
 ?>
 <html>
 <head>
-    <title><?php echo text($openemr_name) . " " . xlt('Login'); ?></title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
hi @mdsupport ,
We have this used in several places without the `shrink-to-fit=no` clause. What is that meant to do?
-brady

Reply:
The tags come from [BS documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/#starter-template).
Per discussion [here](https://stackoverflow.com/questions/33767533/what-does-the-shrink-to-fit-viewport-meta-attribute-do), the tag itself seems to be related to Safari.  Guessing that BS is avoiding potential conflict with browser for consistently responsive behavior.